src/content/blog/ja/ai-driven-development-workflow.md

```
@@ -0,0 +1,68 @@
+
+CodeRabbitによる自動コードレビューにより、手動レビュー時間を80%削減できました。
+
+## 実測データ
+
+- **開発速度**: 従来の7倍
+- **バグ発生率**: 40%減少
+- **コードレビュー時間**: 80%短縮
+
+## ツールチェーンの詳細
+
+### Core Tools
```

markdownlint-cli2 (0.17.2)

Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)